cli/determined_cli/cli.py

```diff
@@ -50,14 +50,15 @@ def agent_info(t: Dict[str, Any]) -> Union[str, List[str]]:
         return [c["agent"] for c in containers]
 
     tasks = r.json()
-    headers = ["ID", "Name", "Slots Needed", "Registered Time", "Agent"]
+    headers = ["ID", "Name", "Slots Needed", "Registered Time", "Agent", "Priority"]
```

Comment:
q: why does it display a priority column but not a weight one?

Reply:
Adding observability for the fair share scheduler is not in scope for this ticket/project.

Reply:
okay. But it will make the CLI looks confusing. It would be nice to do that before releasing

Reply:
I don't think it's too confusing as we label `N/A` when priority scheduler is not being used.